scripts/build/p-klee.inc

```diff
@@ -213,3 +215,15 @@ get_build_artifacts_klee() {
     echo "${KLEE_BUILD_DIR}"
   )
 }
+
+# Check if the binary artifact is installed
+is_installed_klee() {
+  (
+    setup_build_variables_klee
+    [[ -f "${KLEE_BUILD_DIR}"/.is_installed ]]
```

Comment:
Why don't you check here directly for the `klee` binary?

Reply:
The `.is_installed` is created after all has been built successfully. `klee` is not necessarily the last target in the build process, other targets can still fail.